jest.config.js

```diff
@@ -9,6 +9,7 @@ const ignorePattern =
 module.exports = {
     testPathIgnorePatterns: ignorePattern,
     testEnvironment: "node",
+    testRunner: "jest-jasmine2",
```

Comment:
As per https://jestjs.io/blog/2021/05/25/jest-27#flipping-defaults

Reply:
why? We should use `jest-circus`

Reply:
Actually we rely on jasmine, maybe we can update I will look into it.

https://github.com/webpack/webpack-cli/blob/7dad5817ca3a39a5782411d3629512ac7dc161a5/setupTest.js#L5

Reply:
oh yep, we need to migrate as jasmine will soon be deprecated

Reply:
@snitin315 Let's merge it as is and do `jest-circus` migration in the separate PR

Reply:
Will do 👍